tests/test_checkpointing.py

```diff
@@ -1,32 +1,37 @@
-from examples.checkpointing.checkpointing_example import setup as cp_setup
+from examples.checkpointing.checkpointing_example import CheckpointingExample
 from examples.checkpointing.checkpoint import DevitoCheckpoint, CheckpointOperator
 from examples.seismic.acoustic.acoustic_example import acoustic_setup
 from pyrevolve import Revolver
 import numpy as np
 from conftest import skipif_yask
+import pytest
 
 
 @skipif_yask
-def test_forward_with_breaks():
-    time_order = 2
-    fw, gradop, u, rec_s, m0, src, rec_g, v, grad, _, _, nt, dt = cp_setup((150, 150),
-                                                                           750.0,
-                                                                           (15.0, 15.0),
-                                                                           time_order, 4,
-                                                                           40)
-    cp = DevitoCheckpoint([u])
-    wrap_fw = CheckpointOperator(fw, {'u': u, 'rec': rec_s, 'm': m0, 'src': src, 'dt': dt}
-                                 )
-    wrap_rev = CheckpointOperator(gradop, {'u': u, 'v': v, 'm': m0, 'rec': rec_g,
-                                           'grad': grad, 'dt': dt})
-    wrp = Revolver(cp, wrap_fw, wrap_rev, None, nt-time_order)
-    fw.apply(u=u, rec=rec_s, m=m0, src=src, dt=dt)
-    u_temp = np.copy(u.data)
-    rec_temp = np.copy(rec_s.data)
-    u.data[:] = 0
+@pytest.mark.parametrize('space_order', [4])
```

Comment:
did you mean do parametrise this?

Reply:
Although I added it as a parameter, checkpointing is completely orthogonal to space order, so I left it at a single value that can easily be extended if/when someone sees the need. The interesting parametrisation here would be between different equations that access different offsets in time. That's still a couple of PRs away.